reports/summary-2021.md

```diff
@@ -57,33 +57,21 @@ For more, see:
 
 ## Contributor snapshot
 
-62000   
-contributors all time  
-10000  
-new contributors this year  
-  
-1   
-new sig SIG K8s Infra, converted from WG  
-  
+As recorded in [devstats] and [sigs.yaml changes], in 2021 the Kubernetes project had:
 
-1   
-new working group  
-  
-5   
-new chairs and tech leads   
-  
-10 or less  
-unique reviewers in 8 groups  
-  
-8.29   
-average active meeting participants in each group  
-  
-~70000   
-slack members in SIG/WG rooms  
+- `62000` contributors all time
+- `10000` new contributors this year
+- `1` new sig SIG K8s Infra, converted from WG
+- `1` new working group Structured Logging Working Group
+- `5` new chairs and tech leads
+- `10 or less` unique reviewers in 8 groups
+- `8.29` average active meeting participants in each group  
+- `~70000` Slack members in SIG/WG rooms  
```

Comment:
is the backtick-quoting the best way to render these numbers? they seem to read ok without that styling

Reply:
I'll remove the backtick-quote